test/integration/install_scripts_int_test.go

```diff
@@ -110,28 +113,43 @@ func (suite *InstallScriptsIntegrationTestSuite) TestInstall() {
 
 			if tt.Activate != "" || tt.ActivateByCommand != "" {
 				cp.Expect("Creating a Virtual Environment")
-				cp.Expect("Quick Start", termtest.OptExpectTimeout(time.Second*60))
+				cp.Expect("Quick Start", termtest.OptExpectTimeout(time.Minute*2))
 				// ensure that shell is functional
 				cp.ExpectInput()
 
 				cp.SendLine("python3 -c \"import sys; print(sys.copyright)\"")
 				cp.Expect("ActiveState")
 			}
 
-			cp.SendLine("state --version")
+			// We get the default install path and use that to directly invoke
+			// the state tool. This is to avoid inadvertently using the state
+			// tool that is already on the PATH.
+			installPath, err := installation.InstallPathForBranch(constants.BranchName)
+			suite.NoError(err)
+
+			binPath := filepath.Join(installPath, "bin")
+			statePath := filepath.Join(binPath, "state"+osutils.ExeExt)
```

Comment:
e2e should handle this, and if it's not it'll need to be fixed there. You should be able to use `ts.Exe`.

But that is too much here. This shell should have no awareness of the CI State Tool. It sounds like we're bleeding the state tool PATH from the CI environment into our test environment. We should focus on fixing that because there's no way we'll always remember this caveat.